### PR TITLE
feat(Modal): add "bypass_invalidation_selectors" property 

### PR DESCRIPTION
**Pull request by tujoworker**
This property can be used to omit element invalidation outside of a modal.

From the docs:

> Define an array with HTML class selectors (`['.element-selector']`) which should not get invalidated when the modal opens/closes. Use this in order to let some parts of your site still be accessible by screen readers.

### CODE REVIEW

File: packages/dnb-eufemia/src/components/modal/types.ts [hunk 1]
```diff
@@ -307,6 +307,11 @@ export interface ModalContentProps {
    */
   overlay_class?: string
 
+  /**
+   * A list of HTML class selectors that not should get invalidated during modal opening.
```
Comment (langz) [inline packages/dnb-eufemia/src/components/modal/types.ts:311]:
```suggestion
   * A list of HTML class selectors that should not get invalidated during modal opening.
```

File: packages/dnb-eufemia/src/components/modal/types.ts [hunk 2]
```diff
@@ -307,6 +307,11 @@ export interface ModalContentProps {
...
+   */
+  bypass_invalidation_selectors?: Array<string>
```
Comment (langz) [inline packages/dnb-eufemia/src/components/modal/types.ts:313]: Should we include a description for this prop in some .md doc?
Would help for my understanding of what this is used for 🙏

Comment (tujoworker) [reply packages/dnb-eufemia/src/components/modal/types.ts:313]: Yes. Will add it. Thanks for the heads up 🙏

**Pull request merged**